interop/Cargo.toml

```
@@ -25,6 +29,10 @@ tokio-stream = "0.1"
 tonic = {path = "../tonic", features = ["tls-ring"]}
 tower = "0.5"
 tracing-subscriber = {version = "0.3"}
+grpc = {path = "../grpc"}
+protobuf = { version = "4.31.1-release"}
```

```suggestion
protobuf = { version = "4.31.1-release" }
```

We should probably set a toml style and enforce it via some toml formatter if that exists. No need to do anything now just thinking out loud.